wordpress-mobile/WordPress-iOS
# [All Time Widget] Add default widget with supporting files and settings.

Ref #13030 

This is step 1 of adding the 'All Time' stat widget. That is, adding a default widget with supporting files and settings.

To test:
The main purpose of this is to ensure supporting files/settings are correct.
- `Signing & Capabilities`:
  - Uses the Bundle Identifier `org.wordpress.WordPressAllTimeWidget`.  
  - Uses the `WordPress All Time Widget` Provisioning Profiles.
- `*.entitlements`: all the Entitlements looked exactly the same for all widgets & extensions, so I simply copied that implementation for this widget.
- `Info*.plist`s - the only difference between the three is the `Bundle display name`. 
- `Localizable.strings` - is added for Base. It's empty for now. I'll add content when the widget actually does something.

Please review anything you can think of related to widget settings.

In addition (of course), add the widget to your Today view. Since I've simply added the default widget, you'll be dazzled by the ubiquitous `Hello World` message.

<img width="400" alt="default_widget" src="https://user-images.githubusercontent.com/1816888/70099988-a94c3e00-15ed-11ea-9a9b-6d1ee853338b.png">

Footnote: I didn't see a way to "feature flag" a widget, so the feature branch `feature/13030-all_time_widget` will contain all widgety goodness until it's done.

PR submission checklist:
